### PR TITLE
Readme Improvements

### PR DESCRIPTION
**Pull request by rfelber**
Improving the scanners and hooks readme files

### CODE REVIEW

File: scanners/sslyze/README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title: "SSLyze"
 path: "scanners/sslyze"
 category: "scanner"
+type: "Kubernetes"
```
Comment (nigthknight) [inline scanners/sslyze/README.md]: I'm not quite sure why the *type* needs to be _Kubernetes_ and cannot be more general

File: hooks/declarative-subsequent-scans/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+Installing the Cascading Scans hook will add a ReadOnly Hook to your namespace which looks for matching _CascadingRules_ in the namespace and start the according scans. 
+
+```bash
+helm upgrade --install dssh ./hooks/declarative-subsequent-scans/
+```
+
+### Verification
+```bash
+$ kubectl get ScanCompletionHooks
```
Comment (nigthknight) [inline hooks/declarative-subsequent-scans/README.md]: I would recommend removing the **$** at the beginning for more consistency and to make copy and paste more easy

File: hooks/declarative-subsequent-scans/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+NAME   TYPE       IMAGE
+dssh   ReadOnly   docker.io/scbexperimental/hook-declarative-subsequent-scans:latest
+```
+
+## CascadingScan Rules
+The CascadingRules are included directly in each helm chart of the individual scanners. 
+
+```bash
+# Check your CascadingRules
+$ kubectl get CascadingRules
```
Comment (nigthknight) [inline hooks/declarative-subsequent-scans/README.md]: I would recommend removing the **$** at the beginning for more consistency and to make copy and paste more easy

File: hooks/declarative-subsequent-scans/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    matchLabels:
+      securecodebox.io/intensive: light
+EOF
+```
+
+This Scan will used all CascadingRules which are labeled with a "light" intensity.
+You can lookup which CascadingRules this selects by running:
+
+```bash
+$ kubectl get CascadingRules -l "securecodebox.io/intensive=light"
```
Comment (nigthknight) [inline hooks/declarative-subsequent-scans/README.md]: I would recommend removing the **$** at the beginning for more consistency and to make copy and paste more easy

File: scanners/sslyze/README.md (hunk 1 above)
Comment (J12934) [reply scanners/sslyze/README.md]: I think this is a typo / copy paste error.
Type should have probably been "SSL". (probably better to call it tls...)
And state should be "released"

Comment (rfelber) [reply scanners/sslyze/README.md]: yep, that a typo - thanks for finding it :)

**Review by nigthknight — APPROVED** [review]

**Pull request merged**